choice/src/bootstrap.tsx

```diff
@@ -0,0 +1,6 @@
+import App from './App';
+import { createRoot } from 'react-dom/client';
+
+const container = document.getElementById('root');
+const root = createRoot(container!);
+root.render(<App />);
```

Comment:
Just like with the others, new line here would be nice, but totally optional :D

Reply:
Ugh my bad, will follow up in #14